test/tenancy_helper.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package test
```

Comment:
Would be nice to have some test cases for this file.

Reply:
Done